client/tx/search.go

```diff
@@ -30,24 +30,39 @@ func SearchTxCmd(cdc *codec.Codec) *cobra.Command {
 		Use:   "txs",
 		Short: "Search for all transactions that match the given tags.",
 		Long: strings.TrimSpace(`
-Search for transactions that match the given tags. By default, transactions must match ALL tags
-passed to the --tags option. To match any transaction, use the --any option.
-
-For example:
-
-$ gaiacli tendermint txs --tag test1,test2
+Search for transactions that match the given tags. For example:
```

Comment:
What about the `--any` semantics. Has that changed? I think we should clarify in both the short and long descriptions if the search matches based on an ALL or ANY basis.

Reply:
the `--any` flag wasn't being used at all, so I deleted it. Should I change the description to `Search for transactions that match exactly the given tags`

Reply:
done